node/actors/bft/src/lib.rs

```diff
@@ -87,10 +90,17 @@ impl Config {
 
             tracing::info!("Starting consensus actor {:?}", cfg.secret_key.public());
 
-            // This is the infinite loop where the consensus actually runs. The validator waits for either
-            // a message from the network or for a timeout, and processes each accordingly.
-            loop {
-                let InputMessage::Network(req) = pipe.recv.recv(ctx).await?;
+            #[tracing::instrument(skip_all)]
```

Comment:
sry for this being kind of late, but explicit argument passing is in these "*_iter" functions, I actually like the async "{ ... }.instrument(...)" better